runs/runs.yml

```diff
@@ -0,0 +1,5 @@
+pyladies-brno-jaro-po:
+  link: pyladies-brno-jaro-po
+  title: Začátečnický kurz Pyladies
+  description: Naučte se Python vážně od začátku. Žádné předchozí znalosti nejsou potřeba.
+  year: 2017
```

Comment:
The `link` is redundant, the same info is already in the dict key.  I think we need to put the year in the key as well, though, so `2018/pyladies-brno-jaro-po` can use the same scheme.

Perhaps make it a multi-level dict?

    2017:
        pyladies-brno-jaro-po:
            title: ...
            description: ...